File: basic stuff/leetcode/leet_code_344.js
var reverseString = function(s) {
  let end = s.length - 1;
  for (let start = 0; start < end; start++) {
      const temp = s[start];
      s[start] = s[end];
      s[end] = temp;
      end -= 1;
  }
};

File: basic stuff/leetcode/leet_code_344.py
def reverseString(string):
  end = len(string) - 1
  start = 0
  split = list(string)
  while start < end:
    temp = split[start]
    split[start] = split[end]
    split[end] = temp
    start += 1
    end -= 1

  return ''.join(split)